### PR TITLE
Added Starburst Content

### PR DESCRIPTION
**Pull request by taneem-ibrahim**
PTAL @jeff-phillips-18  Thank you!

### CODE REVIEW

**Comment by taneem-ibrahim** [comment]
Added pachyderm details on the product offering along with a link to the pachyderm enterprise offering.

File: data/applications/starburst.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ spec:
   displayName: Starburst Enterprise
   provider: Starburst
   description: |-
-    Unlocks the value of all data by making it fast and easy to access anywhere.
+    Starburst Galaxy is a fully-managed service to run high performance queries across your various data sources using SQL
   kfdefApplications: []
   route: ''
   img: images/starburst.svg
   docsLink: https://docs.starburst.com/
```
Comment (jeff-phillips-18) [inline data/applications/starburst.yaml:11]: This link doesn't work

File: data/getting-started/pachyderm.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Pachyderm
+
+[Pachyderm](https://www.pachyderm.com/platform/#enterprise) is an enterprise-grade, open source data science platform that makes explainable, repeatable, and scalable ML/AI a reality. Our platform brings together version control for data with the tools to build scalable end-to-end ML/AI pipelines while empowering users to use any language/framework/tool they want. Pachyderm has been proven to be the ideal foundation for teams looking to use ML/AI to solve real-world problems in a reliable way. What makes Pachyderm such a natural choice for data science teams is that with just one click, a user can know exactly what data was used to train a model.
```
Comment (jeff-phillips-18) [inline data/getting-started/pachyderm.md:3]: Though we add this, Pachyderm is still `Coming soon` so it won't be seen.

**Comment by taneem-ibrahim** [comment]
@jeff-phillips-18 Fixed the broken docs link for starburst. Thank you for catching this one.

**Review by jeff-phillips-18 — APPROVED** [review]

**Pull request merged**